examples/fully-private-eks-cluster/main.tf

```diff
@@ -14,173 +14,120 @@ provider "kubernetes" {
   }
 }
 
-data "aws_availability_zones" "available" {}
-
 locals {
-  tenant      = var.tenant      # AWS account name or unique id for tenant
-  environment = var.environment # Environment area eg., preprod or prod
-  zone        = var.zone        # Environment with in one sub_tenant or business unit
-  region      = "us-west-2"
+  name   = basename(path.cwd)
+  region = "us-west-2"
+
+  tags = {
+    Blueprint  = local.name
+    GithubRepo = "terraform-aws-eks-blueprints"
+  }
+}
 
-  vpc_cidr     = "10.0.0.0/16"
-  vpc_name     = join("-", [local.tenant, local.environment, local.zone, "vpc"])
-  azs          = slice(data.aws_availability_zones.available.names, 0, 3)
-  cluster_name = join("-", [local.tenant, local.environment, local.zone, "eks"])
+#---------------------------------------------------------------
+# EKS Blueprints
+#---------------------------------------------------------------
+module "eks_blueprints" {
+  source = "../.."
 
-  terraform_version = "Terraform v1.0.1"
+  cluster_name    = local.name
+  cluster_version = "1.21"
+
+  vpc_id             = module.vpc.vpc_id
+  private_subnet_ids = module.vpc.private_subnets
+
+  # Step 1. Set cluster API endpoint both private and public
+  cluster_endpoint_public_access  = true
+  cluster_endpoint_private_access = true
+
+  # Step 2. Change cluster endpoint to private only, comment out the above lines and uncomment the below lines.
+  # cluster_endpoint_public_access  = false
+  # cluster_endpoint_private_access = true
+
+  # EKS MANAGED NODE GROUPS
+  managed_node_groups = {
+    mg_4 = {
```

Comment:
mg_5?

Reply:
Updated in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/551/commits/3335bd466858a2527a90dad8256a0597a0a56262